pkg/orchestration/wiring/k8scompute/plugin.go

```diff
@@ -454,7 +456,7 @@ func buildAffinity(resourceName voyager.ResourceName) *core_v1.Affinity {
 func buildAntiAffinity(resourceName voyager.ResourceName) *core_v1.PodAntiAffinity {
 	matchExpressions := []meta_v1.LabelSelectorRequirement{
 		meta_v1.LabelSelectorRequirement{
-			Key:      "app",
+			Key:      resourceNameLabel,
```

Comment:
resource name isn't unique, is it? we should either prefix it with a service name, or use 2 labels (one for service, another for resource)?
/cc @ash2k

Reply:
Just saw the other thread about PodDisruptionBudget being namespaced, then probably it's fine.

Reply:
Resource name is unique within a namespace, yes. We may still want to just use the labels we set on Pods here anyway for consistency. I think that would be good.

Reply:
@ash2k do you mean the resource name label, and the state label from https://github.com/atlassian/voyager/blob/master/pkg/orchestration/wiring/k8scompute/plugin.go#L248-L251 here?

Reply:
@halcyonCorsair yes, exactly. Perhaps even pass the map with those labels and transform them into the desired shape here.

Reply:
Done.

Reply:
Done.